Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/ActionHandlers/ActUIElementHandler.cs

```diff
@@ -478,5 +489,57 @@ private async Task HandleSetValueOperationAsync()
             string value = _act.GetInputParamCalculatedValue("Value");
             await element.SetAttributeValueAsync(name: "value", value);
         }
+
+        private async Task HandleClickAndValidateAsync()
+        {
+            IBrowserElement elementToClick = await GetFirstMatchingElementAsync();
+            ActUIElement.eElementAction clickType = Enum.Parse<ActUIElement.eElementAction>(_act.GetInputParamValue(ActUIElement.Fields.ClickType).ToString());
+
+            if (clickType == ActUIElement.eElementAction.Click)
```

Comment:
Use function HandleClickOperationAsync to click the element.

Reply:
done